examples/instrumentation-quickstart/otel-collector-config.yaml

```diff
@@ -35,12 +35,26 @@ receivers:
           layout: '%Y-%m-%dT%H:%M:%S.%fZ'
         severity:
           parse_from: body.severity
+          preset: none
+          # parse minimal set of severity strings that Cloud Logging explicitly supports
+          # https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#LogSeverity
+          mapping:
+            debug: debug
+            info: info
+            info3: notice
+            warn: warning
+            error: error
+            fatal: critical
+            fatal3: alert
+            fatal4: emergency
 
-      # set trace_flags to SAMPLED if GCP attribute is set to true
-      - type: add
-        field: body.trace_flags
-        value: "01"
-        if: body["logging.googleapis.com/trace_sampled"] == true
+      # TODO(aaronabbott): uncomment once the log4j2 config is correctly outputing the
+      # logging.googleapis.com/trace_sampled value
+      ## set trace_flags to SAMPLED if GCP attribute is set to true
+      # - type: add
+      #   field: body.trace_flags
+      #   value: "01"
+      #   if: body["logging.googleapis.com/trace_sampled"] == true
```

Comment:
if this returns false, will that just be a no-op? (do we actually need to comment out this config?)

Reply:
Ah good point. I was trying to prevent collector log spam, but it looks like the log spam is actually coming from the `remove` operator:

`2024-04-18T14:52:50.317Z     error   helper/transformer.go:98        Failed to process entry {"kind": "receiver", "name": "filelog", "data_type": "logs", "operator_id": "remove6", "operator_type": "remove", "error": "remove: field does not exist: body['logging.googleapis.com/trace_sampled']", "action": "send", "entry": {"observed_timestamp":"2024-04-18T14:52:50.317546122Z","timestamp":"2024-04-18T14:52:50.214Z","body":{"_exception":{"stackTrace":""},"_logger":"com.example.demo.SingleController","_thread":"reactor-http-epoll-20","logging.googleapis.com/insertId":"2037","logging.googleapis.com/labels":{"span_id":"5aeb502bb6ef209f","trace_flags":"01","trace_id":"5643ea6d8d99b1b30a6fbab66b1a3ddd"},"logging.googleapis.com/sourceLocation":{"function":"com.example.demo.SingleController.handleSingle"},"message":"Going to sleep for 172"},"attributes":{"log.file.name":"app.log"},"severity_text":"INFO","span_id":"WutQK7bvIJ8=","trace_id":"VkPqbY2ZsbMKb7q2axo93Q==","severity":9,"scope_name":""}}`

I'll just let it be for now and reset this file